src/OrchardCore.Modules/OrchardCore.Lists/Feeds/ListPartFeedDisplayDriver.cs

```diff
@@ -31,8 +31,10 @@ public override IDisplayResult Edit(ListPart part)
 
         public override async Task<IDisplayResult> UpdateAsync(ListPart part, IUpdateModel updater)
         {
-            var model = new ListFeedEditViewModel();
-            model.ContentItem = part.ContentItem;
+            var model = new ListFeedEditViewModel
+            {
+                ContentItem = part.ContentItem
```

Comment:
,

Reply:
Okay done and in all other places in this PR